services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/repeated_feedback/prompts/2024_08_28_gemini2.md

```diff
@@ -0,0 +1,6 @@
+One at a time, compare the entry to each numbered entry in the list. Is the entry identical to any item in the list? If so, return {'repeat_feedback' : true}. Otherwise, return {'repeat_feedback' : false}
```

Comment:
This is the template used. Note, including examples in the prompt gave much worse performance.

Reply:
The language does make me wonder how identical "identical" is.  I presume we don't want actual identical comparisons because if we did this could just be a pure Ruby comparison, but I figured I'd drop a note just in case this really should be a Ruby comparison because we mean "identical" literally.

Reply:
That is surprising

Reply:
We don't want identical, but 'identical' seems to work well for the prompt to do a 'semantic match'. It's definitely inaccurate from a strict definition perspective though.

Long-term, this might be a something that doesn't need an LLM (but for not it's performing well).